snappy/snapp.go

```diff
@@ -1757,6 +1762,7 @@ func setUbuntuStoreHeaders(req *http.Request) {
 	req.Header.Set("X-Ubuntu-Frameworks", strings.Join(addCoreFmk(frameworks), ","))
 	req.Header.Set("X-Ubuntu-Architecture", string(Architecture()))
 	req.Header.Set("X-Ubuntu-Release", release.String())
+	req.Header.Set("X-Ubuntu-Core-Wire-Protocol", UbuntuCoreWireProtocol)
```

Comment:
Sorry, I know this was my own suggestion, but can we please drop the "-Core-" part? Looking here in context and thinking about uses for other future products that will depend on this, I think it's more correct.

With that, LGTM. Thanks!